docs/process.md

```diff
@@ -92,6 +92,9 @@ How:
    [emscripten-releases][releases_repo] repo. When this CL is committed, let the
    resulting SHA be `<LTO-sha>`. An example of this CL is
    https://chromium-review.googlesource.com/c/emscripten-releases/+/3781978.
+   After landing the CL, wait for a couple hours before proceeding because
+   building and creating a binary for the new commit will take some time. If the
+   binary is not ready, `create_release.py` in the next step will fail.
```

Comment:
Maybe advise checking `https://ci.chromium.org/p/emscripten-releases/g/main/console`  to see when the commit has been built

Reply:
Done